common/transport/lwm2m/src/main/java/org/thingsboard/server/transport/lwm2m/server/DefaultLwM2mTransportService.java

```diff
@@ -99,9 +99,13 @@ private void startLhServer() {
 
     @PreDestroy
     public void shutdown() {
-        log.info("Stopping LwM2M transport server!");
-        server.destroy();
-        log.info("LwM2M transport server stopped!");
+        try {
+            log.info("Stopping LwM2M transport server!");
+            server.destroy();
+            log.info("LwM2M transport server stopped!");
+        } catch (Exception e) {
+            log.error("", e);
```

Comment:
log.error("Failed to gracefully stop the LwM2M service", e);

Reply:
Ok